pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -371,6 +377,24 @@ func (b *serviceBinder) setApplicationObjects(
 	sbrStatus.Applications = boundApps
 }
 
+// set default value for application selector
+func ensureDefaults(applicationSelector *v1alpha1.ApplicationSelector) {
+	if applicationSelector.LabelSelector == nil {
+		applicationSelector.LabelSelector = &metav1.LabelSelector{}
+	}
+	if applicationSelector.BindingPath == nil {
+		applicationSelector.BindingPath = &v1alpha1.BindingPath{
+			ContainersPath: defaultPathToContainers,
+		}
+	} else {
+		if applicationSelector.BindingPath.ContainersPath == "" && applicationSelector.BindingPath.SecretPath != "" {
```

Comment:
Is `bindingPath.containersPath`'s default value dependent on the presence of `bindingPath.secretPath`? Should it be?

Reply:
I am trying to set the default if container path is not specified but there is secret path given.